test/Dapr.Actors.Test/Runtime/ActorRuntimeTests.cs

```diff
@@ -46,6 +46,17 @@ public void TestExplicitActorType()
             Assert.Contains(RenamedActorTypeName, actorRuntime.RegisteredActorTypes, StringComparer.InvariantCulture);
         }
 
+        [Fact]
+        public void TestRegisterOtherTypeAtRuntime()
+        {
+            var actorType = typeof(ActorRuntimeTests);
+            var actorRuntime = new ActorRuntime();
+
+            actorRuntime.RegisterActor(actorType);
```

Comment:
Can we check that it throws InvalidArgumentException as ActorRuntimeTests is not deriving from Actor?